fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -115,6 +115,22 @@ async def get_fee_6(address: str, contract: Contract, factory_contract: Contract
     """
     return await factory_contract.caller.pairFee(address)
 
+async def get_fee_7(address: str, contract: Contract, factory_contract: Contract) -> float:
```

Comment:
TODO: pull from dev, then fix this to `_ get_fee_7... -> int`.

Reply:
Even though the fee is unlikely to change we will need to fetch it from the CORE contract (i.e. this is different to the Factory contract) 
https://lineascan.build/address/0x8D58ee2D23f7920eA32e534aAD8d6753c88bc01A